command/bridge/premine/params.go

```diff
@@ -0,0 +1,104 @@
+package premine
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"math/big"
+
+	bridgeHelper "github.com/0xPolygon/polygon-edge/command/bridge/helper"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	validatorHelper "github.com/0xPolygon/polygon-edge/command/validator/helper"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+var errMandatoryRootPredicateAddr = errors.New("root erc20 predicate address not defined")
+
+const (
+	nonStakedAmountFlag    = "non-staked-amount"
+	stakedAmountFlag       = "staked-amount"
```

Comment:
WDYT, or maybe not, not sure myself either :thinking_face:
```suggestion
	premineAmountFlag    = "premine-amount"
	stakeAmountFlag       = "stake-amount"
```

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/49/commits/4fbda926da625bc331636ac6f3c059cd75f1dd80